probe-rs/src/debug/snapshots/probe_rs__debug__debug_info__test__atsamd51p19a__static_variables.snap

```diff
@@ -773,4 +773,691 @@ Child Variables:
                               - Typedef: __int8_t
                               - Base: signed char
                   value: "0"
-
+    - name:
+        Named: nested_array
+      type_name:
+        Array:
+          item_type_name:
+            Array:
+              item_type_name:
+                Modified:
+                  - Typedef: uint8_t
+                  - Modified:
+                      - Typedef: __uint8_t
+                      - Base: unsigned char
+              count: 11
+          count: 6
+      value: "uint8_t[11][6] = [\n\tuint8_t[11] = [\n\t\t0,\n\t\t0,\n\t\t0,\n\t\t0,\n\t\t0,\n\t\t0,\n\t\t0,\n\t\t0,\n\t\t0,\n\t\t0,\n\n\t\t... and 1 more\n\t],\n\tuint8_t[11] = [\n\t\t0,\n\t\t0,\n\t\t0,\n\t\t0,\n\t\t0,\n\t\t0,\n\t\t0,\n\t\t0,\n\t\t0,\n\t\t0,\n\n\t\t... and 1 more\n\t],\n\tuint8_t[11] = [\n\t\t0,\n\t\t0,\n\t\t0,\n\t\t0,\n\t\t0,\n\t\t0,\n\t\t0,\n\t\t0,\n\t\t0,\n\t\t0,\n\n\t\t... and 1 more\n\t],\n\tuint8_t[11] = [\n\t\t0,\n\t\t0,\n\t\t0,\n\t\t0,\n\t\t0,\n\t\t0,\n\t\t0,\n\t\t0,\n\t\t0,\n\t\t0,\n\n\t\t... and 1 more\n\t],\n\tuint8_t[11] = [\n\t\t0,\n\t\t0,\n\t\t0,\n\t\t0,\n\t\t0,\n\t\t0,\n\t\t0,\n\t\t0,\n\t\t0,\n\t\t0,\n\n\t\t... and 1 more\n\t],\n\tuint8_t[11] = [\n\t\t0,\n\t\t0,\n\t\t0,\n\t\t0,\n\t\t0,\n\t\t0,\n\t\t0,\n\t\t0,\n\t\t0,\n\t\t0,\n\n\t\t... and 1 more\n\t]]"
```

Comment:
This is 6 counts of 11-element arrays. We should print them as `uint8_t[6][11]` as the variable is declared (`uint8_t nested_array[6][11]`). C writes nested arrays nesting from right to left, no wonder I can never remember how they are ordered.

Note to self: initialize the array with data, we don't know if the elements are arranged correctly.